CHANGELOG.md

```diff
@@ -2,9 +2,14 @@
 
 ## XX.XX.XX - 2023-XX-XX
 
+## 20.21.2 - 2023-04-10
+
 ### All SDKs
 * [CHANGED][6492](https://github.com/stripe/stripe-android/pull/6492) Updated Compose to 1.4.1.
 
+### Financial Connections
```

Comment:
Also think we should keep the existing ordering of `Payments` > `PaymentSheet` > `Financial Connections` > …